### DISCUSSION TITLE
工程无法识别插件包

### DISCUSSION THREAD

**Discussion by RizomaDong** [endink/Mediapipe4u-plugin#161] [Q&A]
我在之前已经按照您的教学视频搭建好工程并且运行成功了，但是目前工程打不开了，提示没有插件，非常奇怪，重新创建工程也不识别M4U插件包，
![屏幕截图 2024-04-28 124348](https://github.com/endink/Mediapipe4u-plugin/assets/132998142/3fad95c5-4f87-4976-92a5-e13b2b320a21)

**Comment by endink** [comment]
最新版已经下架LLM，需要 LLM 请使用旧版
